### ISSUE TITLE
[rush] console.log swallowed by custom command

### ISSUE THREAD

**Issue by knownasilya** [microsoft/rushstack#2021]
**Is this a feature or a bug?**

- [ ] Feature
- [x] Bug

**Please describe the actual behavior.**

I have a custom rush command, `watch`:

```json
{
      "commandKind": "bulk",
      "name": "watch",
      "summary": "Watch all repos with watch script",
      "enableParallelism": true,
      "ignoreMissingScript": true
    }
```

Which logs cli output from the `debug` module, but any `console.log` output is not in the terminal when the custom command runs.

**What is the expected behavior?**

Log `console.log` output.

* **Tool:** rush
* **Tool Version:** 5.27.3 
* **Node Version:** 14.5.0
  * **Is this a LTS version?** no
  * **Have you tested on a LTS version?** no
* **OS:** macos 10.15.5

**Comment by iclanton** [comment]
You should be able to use the `--verbose` flag, which will proxy each project's output to the console.

However, Rush currently doesn't support watch particularly well. We print one project's logs at a time for readability, which involves waiting for the process to terminate before writing the next project. If you're looking to do multi-project watch, you'll probably only see logs from the first project that starts printing to the console.

This is something that I believe @octogonz is exploring improving in the near future.

**Comment by knownasilya** [comment]
Good to know, about both cases.

**Comment by octogonz** [comment]
@knownasilya Should we close this issue?

**Comment by knownasilya** [comment]
Seems like a bad default for custom commands

**Comment by octogonz** [comment]
Good point - let's fix that. @knownasilya are you interested to make a PR?

**Comment by octogonz** [comment]
As far as design, some possible ideas would be:
- default to `--verbose` for all commands
- just for custom commands
- or maybe allow it to be a separate configurable setting for each command?

If it is verbose by default, then it seems we need a CLI parameter that is the opposite of `--verbose` if the person wants to make it non-verbose.

**Comment by unional** [comment]
Hi, I faced the same problem.

I think one way to fix that is to allow custom commands to specify `--verbose` by default.

My use case is `rush coverage`.
I want to see the resulting code coverage when the command completes.

Right now I have to do `rush coverage -v` or `rush coverage -o <pkg> -v`
